### PR TITLE
Argentina

### PR DESCRIPTION
**Pull request by fojackson8**
Fixing convert_location to deal with CABA and limitCountry to Argentina

### CODE REVIEW

File: ingestion/functions/parsing/argentina/argentina.py [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+    if entry['carga_provincia_nombre']:
+        if entry['carga_provincia_nombre'] == 'CABA':
+            return {
+                "query": "Buenos Aires, Argentina",
+                "limitToCountry": "Argentina"}
+        else:
+
+            return {
+                "query": f"{entry['carga_provincia_nombre']}, Argentina",
+                "limitToCountry": "Argentina"}
```
Comment (attwad) [inline ingestion/functions/parsing/argentina/argentina.py:59]: you can't specify limitToCountry yet, it hasn't been implemented https://github.com/globaldothealth/list/issues/1306

Last part of this issue should be to go through all the parsers and add that country restriction to them, you can ignore for now.

**Review by attwad — COMMENTED** [review]
Not sure why all these files are showing up as new, only the diff should show up otherwise it makes reviewing what's changed impossible, perhaps try a new branch instead next time?

File: ingestion/functions/parsing/argentina/argentina.py (hunk 1 above)
Comment (fojackson8) [reply ingestion/functions/parsing/argentina/argentina.py:59]: ok will make new branch and remove limitToCountry, thanks

Comment (attwad) [reply ingestion/functions/parsing/argentina/argentina.py:59]: not done?

**Review by attwad — COMMENTED** [review]
if it gets too confusing to fix the branch/git issue and merge conflicts, perhaps just try to start from scratch (new branch from main)

**Pull request merged**